lib/constellation.rb

```diff
@@ -0,0 +1,39 @@
+class ConstellationHttpException < StandardError
+end
```

Comment:
Let’s do it in another PR then, we already had to use the cute puppy GIF once this PR.